Adds a few variants

GregLMcDonald opened the pull request:
C'est drôle en crisse, Henry.  Made my day. Here's a few additional variants from the Mauricie region (or maybe just my in-laws).

GregLMcDonald committed:
Update index.ts

Additional variants from the Mauricie region (or maybe just my in-laws)

henrymyers commented on lib/index.ts:
```diff
@@ -13,7 +13,7 @@ function getAllSwears(): Array<swearVariants> {
         [`calvaire`, `calvince`, `calvinouche`],
         [`mosus`],
-        [`maudit`, `mautadit`, `maudine`],
+        [`maudit`, `mautadit`, `maudine`, `mautadine`],
```
👍

henrymyers commented on lib/index.ts:
```diff
@@ -29,7 +29,7 @@ function getAllSwears(): Array<swearVariants> {
         [`viande à chien`],
         [`bout d'crisse`],
-        [`cul`],
+        [`cul`, `sainte-fesse`],
```
Drôle, mais la seule référence que je trouve en ligne c'est une série télé
<img width="741" alt="Screen Shot 2020-10-29 at 7 35 12 PM" src="https://user-images.githubusercontent.com/942003/97643461-12b31780-1a1e-11eb-919c-c541c6ce6956.png">

C'est peut-être pas assez commun?

GregLMcDonald replied:
Si on y va plutôt avec le pluriel, "saintes-fesses", il y en plus:

C’est ainsi qu’après plusieurs heures de remue-méninges avec mon fils, nous en sommes venus à des jurons comme «saintes-fesses» pour Simone, comme «jérusalem» pour Claude, «crotte de bouc» et «ostin d’beu» pour Zoé et Renaud, «apostrophe» pour Benoît Girard et «sainte-bénite» pour Juliette Huot.
(https://www.journaldemontreal.com/2015/02/20/en-francais-sil-vous-plait)

Au revoir «Citron!», «Saintes fesses!», «Sacrament, Ginette!», «Géritol!» et autres expressions colorées nées de la plume des auteurs, popularisées à la télé et adoptées dans le langage populaire!
(http://207.253.106.223/divertissement/telemedias/nouvelles/archives/2016/06/20160608-113639.html#2)

Saintes fesses que c’est bon. C’est ça qui arrive quand 5 petits chums de Portland (avec des membres de YOB) se mettent ensemble pour faire de la musique.
https://boulevardbrutal.com/2017/03/29/norska-va-faire-allonger-ta-petite-barbe/

Saintes Fesses ! Je me pose des questions des fois. Une amie esthéticienne me disait que des clients demandent des blanchiments d'anus à la clinique.
(https://www.jasez.ca/forums/topic,19917-drole_d_idee.html)

GregLMcDonald replied:
J'ai changé pour le pluriel. In other words "Holy Butt Cheeks!" rather than "St. ButtCheek" 😁

GregLMcDonald committed:
Update index.ts

Turns out, the plural form is the more common form of this rather cute and light swear.

GregLMcDonald committed:
Update index.ts

henrymyers commented on lib/index.ts:
```diff
@@ -29,7 +29,7 @@ function getAllSwears(): Array<swearVariants> {
         [`viande à chien`],
         [`bout d'crisse`],
-        [`cul`],
+        [`cul`, `saintes fesses`],
```
Merci pour les références! Ça va donner des beaux résultats 😝 

```
Saintes fesses de Jésus Marie Joseph
```

henrymyers committed:
Merge branch 'master' into master
